--- system-model.cfg.xml	2014-06-23 11:04:21.462690535 -0500
+++ system-model.cfg.xml_NEW	2014-06-23 11:09:01.730700516 -0500
@@ -13,8 +13,8 @@
       <!--
       <filter name="ip-identity" />
       <filter name="header-identity" />
-
       -->
+      <filter name="header-translation" />
       <filter name="rate-limiting" />
     </filters>
       <services>
@@ -22,7 +22,7 @@
       </services>
     <destinations>
       <!-- Update this endpoint if you want Repose to send requests to a different service -->
-      <endpoint id="open_repose" protocol="http" hostname="openrepose.org" root-path="/" port="80" default="true"/>
+      <endpoint id="local_httpbin" protocol="http" hostname="localhost" root-path="/" port="8000" default="true"/>
     </destinations>
   </repose-cluster>
 </system-model>
